bindings/python/pymongoarrow/bson_util.pyx

```diff
@@ -0,0 +1,33 @@
+# distutils: language=c++
+# cython: language_level=3
+
+from libcpp cimport bool as cbool
+
+from pymongoarrow.libbson cimport *
+
+
+cdef list _docs_from_bson_stream(const uint8_t *docstream, size_t length):
+    cdef bson_reader_t *stream_reader = bson_reader_new_from_data(docstream, length)
+
+    cdef cbool reached_eof = False
+    cdef const bson_t *doc = bson_reader_read(stream_reader, &reached_eof)
+    cdef bson_iter_t doc_iter
+
+    documents = []
+
+    while doc != NULL:
+        if not bson_iter_init(&doc_iter, doc):
+            raise RuntimeError
```

Comment:
Please add a helpful error message to this RuntimeError.

Reply:
bson_util has been removed.